src/coreclr/System.Private.CoreLib/src/System/Reflection/TypeNameParser.CoreCLR.cs

```diff
@@ -80,6 +84,54 @@ namespace System.Reflection
             }.Parse();
         }
 
+        internal static RuntimeType GetTypeByNameUsingCARules(string typeName, RuntimeModule scope)
+        {
+            ArgumentException.ThrowIfNullOrEmpty(typeName);
```

Comment:
The native implementation defines the argument name to be `className`, does that matter in this case?

Reply:
I do not think that we care about the exact argument name that shows up in the error message in this case. `typeName` is the most correct and consistent name.